internal/servers/server.go

```diff
@@ -298,6 +297,22 @@ func (s *Container) Run(
 					},
 				},
 			}),
+			runtime.WithMiddlewares(func(next runtime.HandlerFunc) runtime.HandlerFunc {
```

Comment:
Need to run the otelhttp handler as a middleware so that it can read the path from the mux's context.